backend/EasyMeets.Core/EasyMeets.Core.DAL/Context/ModelBuilderExtensions.cs

```diff
@@ -21,6 +20,7 @@ public static void Configure(this ModelBuilder modelBuilder)
             modelBuilder.ApplyConfigurationsFromAssembly(typeof(ExternalAttendeeConfig).Assembly);
             modelBuilder.ApplyConfigurationsFromAssembly(typeof(MeetingConfig).Assembly);
             modelBuilder.ApplyConfigurationsFromAssembly(typeof(QuestionsConfig).Assembly);
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(EmailTemplateConfig).Assembly);
```

Comment:
Please delete this

Reply:
Why we should delete config for EmailTemplate entity ?

Reply:
Because it is a reducer